### ISSUE TITLE
easylist: e11956.x.akamaiedge.net blocks whole aliexpress.com

### ISSUE THREAD

**Issue by MaDill** [RPiList/specials#1720]
easylist: e11956.x.akamaiedge.net blocks whole aliexpress.com

assets.alicdn.net
www.aliexpress.com
campaign.aliexpress.com
is.alicdn.com
shoppingcart.aliexepress.com

should be fixed in original easylist: https://github.com/easylist/easylist/issues/19755

**Comment by RPiList** [comment]
Thx for helping us. Domains have been removed. Greetings

**Comment by MaDill** [comment]
When are the updated lists available?

In the current list it is still there: https://github.com/RPiList/specials/blob/4cecb77e3fb36e4ca28e541734f30bb3974217d0/Blocklisten/easylist#L12633

**Comment by RPiList** [comment]
Hi,

thx for helping us.

It already was deleted yesterday, when I wrote it. 

But https://easylist.to/easylist/easylist.txt has the domain listed again, thats why our crawler imported it again.

I removed it again and whitelisted it (will be whitelisted for a few days). I understood they will remove it as well?

Because our list rebuilds itself when updated... so when they removed the domain, it will be removed from our list automatically.

Removal takes immediate effekt on our lists.

Thx for helping us. Greetings
